src/openrct2/ride/Ride.cpp

```diff
@@ -1880,20 +1880,8 @@ static void CircusMusicUpdate(Ride* ride)
  *
  *  rct2: 0x006ABE85
  */
-static void ride_music_update(Ride* ride)
+void DefaultMusicUpdate(Ride* ride)
 {
-    if (ride->type == RIDE_TYPE_CIRCUS)
-    {
-        CircusMusicUpdate(ride);
-        return;
-    }
-
-    const auto& rtd = ride->GetRideTypeDescriptor();
-    if (!rtd.HasFlag(RIDE_TYPE_FLAG_MUSIC_ON_DEFAULT) && !rtd.HasFlag(RIDE_TYPE_FLAG_ALLOW_MUSIC))
```

Comment:
To me it looks like there are 3 settings Default, Circus and None.

Reply:
It was an error from me, this check is necessary.